File: binaryToDecimal.py
# Binary To Deciaml Converter in Python
import  math
binary = input("Enter binary number : 011... ");
decimal = 0;
i = len(binary)-1;
while i > -1:
    if "1" == binary[(len(binary)-1) - i]: decimal += math.pow(2,i);
    i-=1;
print(f"{binary} = {int(decimal)}");

File: binaryToDecimal.c
// Binary To Deciaml Converter in C
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>
int main(void){
    char *binary = NULL;
    size_t n = 0;
    printf("Enter binary number : 011... ");
    getline(&binary, &n, stdin);
    double decimal = 0;
    int i = strlen(binary)-2;
    while (i > -1){
        if('1' == binary[(strlen(binary)-2) - i]) decimal += pow(2, i);
        i--;
    }
    printf("%g", decimal);
    return 0;
}